### PR TITLE
fixing docs related to 3.2.0

### PR DESCRIPTION
**Pull request by shaniR**
### Issues

Fixes #

---
#### Maintainers: Check before merge
- [ ] Manual verification

### CODE REVIEW

**Comment by AmaliMatharaarachchi** [comment]
There are merge conflicting files. Please fix it.

File: en/docs/getting-started/about-this-release.md [hunk 1]
```diff
@@ -62,13 +62,18 @@ WSO2 API Microgateway 3.2.0 is the successor of WSO2 API Microgateway version 3.
...
     
 -   **JWT Revocation with WSO2 API Manager**    
     WSO2 API Microgateway has ability to persist revoked access tokens inside WSO2 API Manager (Traffic Manager). 
     For more information, visit [JWT Token Revocation](how-tos/security/jwt-token-revocation/#jwt-token-revocation).
     
+-   **Support reading open API definition in interceptors**
+   
+-   **Support reading configurations from java interceptors**
+
+-   **Mutual SSL support per API**
```
Comment (AmaliMatharaarachchi) [inline en/docs/getting-started/about-this-release.md:76]: Shall we add a small description of what was improved? mutual SSL per API support was there in 3.1.0 as well.

Comment (shaniR) [reply en/docs/getting-started/about-this-release.md:76]: yes.. we need the descriptions from the team. Please go ahead and add them.

Comment (AmaliMatharaarachchi) [reply en/docs/getting-started/about-this-release.md:76]: @praminda @VirajSalaka @menakaj Would you be able to help in adding a description?

File: en/docs/getting-started/about-this-release.md [hunk 2]
```diff
@@ -62,13 +62,18 @@ WSO2 API Microgateway 3.2.0 is the successor of WSO2 API Microgateway version 3.
...
 
--   **Integrate with Third Party Key Managers**     
+-   **Integrate with multiple Third Party Key Managers**     
     WSO2 API Microgateway 3.2.0,  is able to integrate with other key manager including the WSO2 API Manager Key Manager.
     
 -   **JWT Revocation with WSO2 API Manager**    
     WSO2 API Microgateway has ability to persist revoked access tokens inside WSO2 API Manager (Traffic Manager). 
     For more information, visit [JWT Token Revocation](how-tos/security/jwt-token-revocation/#jwt-token-revocation).
     
+-   **Support reading open API definition in interceptors**
```
Comment (praminda) [inline en/docs/getting-started/about-this-release.md:72]:
```suggestion
-   **Support reading open API definition in interceptors**
    Enables developers to access the OpenApi definitions of the current runtime and implement custom behaviors.
```

File: en/docs/getting-started/about-this-release.md [hunk 3]
```diff
@@ -62,13 +62,18 @@ WSO2 API Microgateway 3.2.0 is the successor of WSO2 API Microgateway version 3.
...
+   
+-   **Support reading configurations from java interceptors**
```
Comment (praminda) [inline en/docs/getting-started/about-this-release.md:74]:
```suggestion
-   **Support reading configurations from java interceptors**
    Provides access to microgateway runtime configurations from java interceptors.
```

File: en/docs/getting-started/about-this-release.md (hunk 1 above)
Comment (praminda) [reply en/docs/getting-started/about-this-release.md:76]:
```suggestion
-   **Mutual SSL support per API**     
    Provides ability to specify which certificates should be trusted by each API. Prior to this improvement, any of the microgateway's trusted certificates could be used to invoke a mutual SSL protected API.
```

**Review by AmaliMatharaarachchi — APPROVED** [review]

**Pull request merged**